goldens/public-api/core/testing/testing.d.ts

```diff
@@ -29,6 +29,12 @@ export declare const ComponentFixtureNoNgZone: InjectionToken<boolean[]>;
 export declare function discardPeriodicTasks(): void;
 
 export declare function fakeAsync(fn: Function): (...args: any[]) => any;
+export declare namespace fakeAsync {
```

Comment:
This should be typed as (delete `function fakeAsync`):
```suggestion
interface FakeAsync {
  // docs here
  (fn: Function): (...args: any[]) =>any;
  
  // docs here
  wrap({
    before?: ()=>void
    after?: ()=>void
  }): FakeAsync;
}
```

Reply:
Got it, this resolves the typescript parsing failure issue in `aio`, thank you!